### PR TITLE
Add step in the SOP

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9678

## What
Add step

### CODE REVIEW

File: SOP/SOP-push.adoc [hunk 1]
```diff
@@ -431,6 +431,7 @@ kind: Secret
 NOTE: The values described above should not be the same but should all data keys shoud be present with each respective value.
 +
 . Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. In order to fix it, try to deploy it again by running oc deploy --latest dc/unifiedpush`
```
Comment (grdryn) [inline SOP/SOP-push.adoc]:
```suggestion
. In order to fix it, try to deploy it again by running `oc rollout --latest dc/unifiedpush`
```

```
$ oc deploy
Error: unknown command "deploy" for "oc"
Run 'oc --help' for usage.
```

File: SOP/SOP-push.adoc [hunk 2]
```diff
@@ -467,6 +468,7 @@ NOTE: You can save the logs by running `oc logs <database-podname> > <filename>.
 . Check if you are able to see any useful information in the logs which can lead you for the root cause of the issue. Also, by capturing the logs you are able to provide a required information for its maintainers if it be required.
 .. Check if the Database image was pulled successfully.
 . Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. In order to fix it, try to deploy it again by running oc deploy --latest dc/unifiedpush-postgresql`
```
Comment (grdryn) [inline SOP/SOP-push.adoc]:
```suggestion
. In order to fix it, try to deploy it again by running `oc rollout --latest dc/unifiedpush-postgresql`
```

**Review by grdryn — CHANGES_REQUESTED** [review]

File: SOP/SOP-push.adoc (hunk 2 above)
Comment (camilamacedo86) [reply SOP/SOP-push.adoc]: 👍

**Comment by camilamacedo86** [comment]
Hi @grdryn changed as you requested.

**Review by grdryn — APPROVED** [review]

**Pull request merged**